src/sentry/relay/config/__init__.py

```diff
@@ -474,7 +474,7 @@ def get_transaction_metrics_settings(
                 assert breakdown_config["type"] == "spanOperations"
 
                 for op_name in breakdown_config["matches"]:
-                    metrics.append(f"d:transactions/breakdowns.ops.{op_name}")
+                    metrics.append(f"d:transactions/breakdowns.span_ops.ops.{op_name}")
```

Comment:
This is how Relay extracts it, see https://github.com/getsentry/relay/blob/bad9c5969800e35dd52e8b702373ab61b348ad2a/relay-server/src/metrics_extraction/transactions.rs#L578.